package.json

```diff
@@ -1,65 +1,47 @@
 {
-  "main": "lib/Main.js",
+  "private": true,
   "license": "MIT",
-  "version": "1.0.0",
   "scripts": {
+    "start": "node lib/Main.js",
     "build": "tsc 1>&2 && npm run copy:test-assets",
-    "clean": "rimraf lib out",
     "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
-    "start": "node ./lib/main.js",
-    "start:snapshot": "node ./lib/MainSnapshot.js",
+    "clean": "rimraf lib",
     "test": "mocha ./lib/tests/*.test.js",
     "copy:test-assets": "cpx \"./src/tests/assets/**/{*,.*}\" ./lib/tests/assets"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/imodeljs/imodel-reporter"
-  },
+  "repository": {},
   "dependencies": {
-    "@bentley/backend-itwin-client": "^2.19.2",
-    "@bentley/bentleyjs-core": "^2.19.2",
-    "@bentley/ecschema-metadata": "^2.19.2",
-    "@bentley/electron-manager": "^2.19.2",
-    "@bentley/geometry-core": "^2.19.2",
-    "@bentley/imodelhub-client": "^2.19.2",
-    "@bentley/imodeljs-backend": "^2.19.2",
-    "@bentley/imodeljs-common": "^2.19.2",
-    "@bentley/presentation-common": "^2.19.2",
-    "@bentley/itwin-client": "^2.19.2",
-    "@bentley/rbac-client": "^2.19.2",
-    "@bentley/telemetry-client": "^2.19.2",
-    "electron": "^11.1.0"
-  },
+    "@itwin/appui-abstract": "4.0.0-dev.66",
```

Comment:
update to the latest dev package.  There is a security issue in xml2js and we hopefully have a new dev build that fixes the issue.  Either way it's good to get up to the latest as we near release